### ISSUE TITLE
Synthese - Onglets Taxonomie, Zonage et Validation vides

### ISSUE THREAD

**Issue by AudreyEnGuyane** [PnX-SI/GeoNature#2802]
Bonjour,

Suite à la mise à jour de taxref v16 (et à une remise au carré des taxons guyanais et statuts des espèces), je me rends compte que les données récemment intégrées dans GeoNature 2.13.3 (via Occtax mobile et Occtax) qui sont consultées dans la synthèse ont des onglets Taxonomie, Zonage et Validation vides (pas testé avec l'outil import).
Ces onglets vides apparaissent pour les données saisies après une date donnée, comme si le lien ne se faisait plus avec ces infos génériques.

Pourtant:
- les vm du schéma taxonomie ont toutes été mises à jour et il y a bien des refs dans la vue bdc_statuts (si c'est bien celle utilisée pour lister les statuts).
- la table cor_area_synthese est bien à jour avec les objets en correspondance avec les données (si c'est bien le contenu de cette table qui est affiché dans  l'onglet "Zonage")
- il y a bien des refs dans la vm_valid_profiles ou dans la vm_cor_taxon_phenology 
- ces infos sont bien affichées pour les données antérieures au 25/09.

Du coup, je ne comprend pas trop d'où ça pourrait venir... J'ai dû passer à côté de quelque chose.

**Comment by camillemonchicourt** [comment]
En effet il me semble que tu as mis à jour ce qu'il faut.
Tu dois avoir des erreurs dans la console du navigateur qui peuvent aider à identifier le soucis, mais sur des erreurs dans les logs du serveur. Sans ça il n'est pas possible d'identifier la source du problème.

**Comment by AudreyEnGuyane** [comment]
On a trouvé!
Il semblerait que ça vienne de la table gn_synthese.defaults_nomenclatures_value: le fait que l'id_organisme soit différent de zéro semble générer des problèmes (bien que ce soit le bon id_organisme) car un enregistrement est créé avec un cd_nomenclature_valid_status = 0.
En mettant tous les id_organisme à 0 (tous les organismes), la valeur par défaut STATUT_VALID est bien prise en compte.
Par contre, il y a quelque chose à revoir si l'administrateur veut attribuer des valeurs par défaut différentes selon les organismes...

**Comment by DonovanMaillard** [comment]
Oui, soit le supprimer, soit l'exploiter en allant au bout de la démarche. Mais entre le fait de laisser le champs null et le fait d'avoir (pour le moment) des nomenclatures "non renseigné" c'est pas là ou c'est le plus utile...

**Comment by camillemonchicourt** [comment]
@AudreyEnGuyane tu avais besoin de définir des valeurs par défaut différentes selon les organismes ??
